src/anemoi/training/train/forecaster.py

```diff
@@ -124,6 +125,23 @@ def __init__(
             config.hardware.num_gpus_per_node * config.hardware.num_nodes / config.hardware.num_gpus_per_model,
         )
 
```

Comment:
Do we actually need any of these here? same for the model_comm_group etc. ... above. I think these are properly initialised by the strategy which uses model.set_model_comm_group and set_reader_group. So might be enough to initialise these to sensible default values when the model is not sharded.

Reply:
I changed it to pass everything from the DDPGroupStrategy via model.set_model_comm_group() and set_reader_group().